### PR TITLE
Modify logic to store slice of slices in the DB.

### LINKED ISSUE #38: Modify logic to store slice of slices in the DB.
Currently, we have `[][]float32` object that stores pairwise similarity - the one must find the way to store it in the DB in the respective implementation of the `SaveClient(...)` in `db_container.go`

### PR DESCRIPTION
**Pull request by gevorgyana**
- Use encoding/json.
- Fix bug (string literal with ' around it)

Closes #38 

Signed-off-by: Artyom Gevorgyan <artemhevorhian@gmail.com>

### CODE REVIEW

File: web-service/api/python_wrappers/similarity_module.py [hunk 1]
```diff
@@ -79,6 +79,18 @@ def heatmap(x_labels, y_labels, values):
         TOKENIZER_OPTION = 2 # stemming
 
     FILE_OBJECTS = (codecs.open(join(INPUT_FOLDER, f), encoding='utf-8').read() for f in FILE_NAMES)
+    # FIXME
+    # File "/home/artyom/basic-online-judge/web-service/api/python_wrappers/similarity_module.py", line 87, in <module>
```
Comment (Pilipets) [inline web-service/api/python_wrappers/similarity_module.py:83]: Please, don't paste bug to the comments - rather create the issue with the proper tag.

File: web-service/api/storage_container/db_container.go [hunk 2]
```diff
@@ -64,15 +65,22 @@ func (self *DbClientContainer) GetValue(id guuid.UUID) (string, bool) {
 	var result string
 	err := self.getStatement.QueryRow(id.String()).Scan(&result)
 	if err != nil {
-		ErrorLogger.Println(err)
```
Comment (Pilipets) [inline web-service/api/storage_container/db_container.go:67]: Why deleting error logging here?

File: web-service/api/storage_container/db_container.go [hunk 3]
```diff
@@ -64,15 +65,22 @@ func (self *DbClientContainer) GetValue(id guuid.UUID) (string, bool) {
...
+	err = json.Unmarshal([]byte(result), &result_internal)
+	if err != nil {
+	        return "", false
+	}
 	return result, true
 }
 
 func (self *DbClientContainer) SaveClient(id guuid.UUID, res [][]float32) {
-	_, err := self.saveStatement.Exec(id.String(), id.String())
+       result_binary, err := json.Marshal(res)
```
Comment (Pilipets) [inline web-service/api/storage_container/db_container.go:79]: use gofmt for the proper indent formatting

File: web-service/api/storage_container/db_container.go [hunk 4]
```diff
@@ -64,15 +65,22 @@ func (self *DbClientContainer) GetValue(id guuid.UUID) (string, bool) {
...
+	if err != nil {
+	        # TODO
```
Comment (Pilipets) [inline web-service/api/storage_container/db_container.go:81]: TODO what?

**Review by Pilipets — CHANGES_REQUESTED** [review]

**Comment by gevorgyana** [comment]
It does not compile but the rest should be fixed with a patch to Python, I suggest so.

File: web-service/api/storage_container/db_container.go (hunk 4 above)
Comment (gevorgyana) [reply web-service/api/storage_container/db_container.go]: removed

File: web-service/api/storage_container/db_container.go (hunk 3 above)
Comment (gevorgyana) [reply web-service/api/storage_container/db_container.go]: fixed

File: web-service/api/storage_container/db_container.go (hunk 2 above)
Comment (gevorgyana) [reply web-service/api/storage_container/db_container.go:67]: fixed

File: web-service/api/python_wrappers/similarity_module.py (hunk 1 above)
Comment (gevorgyana) [reply web-service/api/python_wrappers/similarity_module.py]: done

**Review by Pilipets — APPROVED** [review]

**Pull request merged**